community/elections/2017/README.md

```diff
@@ -58,7 +58,7 @@ Michelle Noorali |	Microsoft |	[@michelleN](https://github.com/michelleN)
 [Phillip Wittrock](pwittrock_bio.md) |	Google | 	[@pwittrock](https://github.com/pwittrock)
 Quinton Hoole	 | Huawei |	[@quinton-hoole](https://github.com/quinton-hoole)
 [Rob Hirschfeld](rhirschfeld_bio.md) |	RackN	| [@zehicle](https://github.com/zehicle)
-Sebastien Goasguen |	Bitnami |	[@sebgoa](http://github.com/sebgoa)
+[Sebastien Goasguen](sebastien_goasguen_bio.md) |	Bitnami |	[@sebgoa](http://github.com/sebgoa)
```

Comment:
@sebgoa It's not a restriction but it'd be good to follow a name convention for bio files. Most of the files names are using lower case firstname+lastname with no dash or underscore between them :) ref: #1058

cc @castrojo